clients/web/test/specRunner.js

```diff
@@ -176,6 +176,16 @@ page.onLoadFinished = function (status) {
         page.evaluate(function () {
             if (window.girderTest) {
                 girderTest.promise.done(function () {
+                    // Allow Jasmine to compare RegExp using toEqual, toHaveBeenCalledWith, etc.
+                    jasmine.getEnv().addEqualityTester(function (a, b) {
```

Comment:
Or alternatively, to pass a custom comparator to a single invocation of `toHaveBeenCalledWith`?